src/test/kotlin/ApplicationTest.kt

```
@@ -696,12 +717,13 @@ class ApplicationTest {
         this.testInfo = testInfo
     }
 
-    private fun cleanUpResources(teamcityClient: TeamcityClassicClient) {
+    private fun cleanUpResources(teamcityClient: TeamcityClassicClient, config: TeamcityTestConfiguration) {
         try {
             teamcityClient.deleteProject(TEST_PROJECT)
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)